app/lib/discourse_automation/scripts/send_pms.rb

```diff
@@ -12,7 +12,7 @@
   field :receiver, component: :user, triggerable: :recurring
   field :sendable_pms, component: :pms, accepts_placeholders: true
 
-  triggerables %i[user_added_to_group stalled_wiki recurring]
+  triggerables %i[user_added_to_group stalled_wiki recurring api_call]
```

Comment:
I actually couldn't get this triggerable to work for API call, because it needs `users` in the context, however you can't pass this via a POST request, the best you can do is JSON array of `usernames`, so I think that needs to be accounted for here:

https://github.com/discourse/discourse-automation/blob/a46f7ec142d4ea4bbd8cd84dd221564b9986b85d/app/lib/discourse_automation/scripts/send_pms.rb#L24-L24

Reply:
explained later